pkg/exchange/ftx/exchange.go

```diff
@@ -203,25 +273,56 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 		return nil, fmt.Errorf("ftx returns failure")
 	}
 
-	var kline []types.KLine
+	var klines []types.KLine
 	for _, r := range resp.Result {
 		globalKline, err := toGlobalKLine(symbol, interval, r)
 		if err != nil {
 			return nil, err
 		}
-		kline = append(kline, globalKline)
+		klines = append(klines, globalKline)
 	}
-	return kline, nil
+	return klines, nil
 }
 
+//resolution field in api
+//window length in seconds. options: 15, 60, 300, 900, 3600, 14400, 86400, or any multiple of 86400 up to 30*86400
 var supportedInterval = map[int]struct{}{
-	15:    {},
-	60:    {},
-	300:   {},
-	900:   {},
-	3600:  {},
-	14400: {},
-	86400: {},
+	15:      {},
+	60:      {},
+	300:     {},
+	900:     {},
+	3600:    {},
+	14400:   {},
+	86400:   {},
+	172800:  {},
+	259200:  {},
+	345600:  {},
+	432000:  {},
+	518400:  {},
+	604800:  {},
+	691200:  {},
+	777600:  {},
+	864000:  {},
+	950400:  {},
+	1036800: {},
+	1123200: {},
```

Comment:
maybe this can be merged into `supportedIntervals`? (I am not sure)

Reply:
yes, we  could do that to limit the options to only supprotedIntervals items.  the issue is the config range different than types.interval .